sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorAdministrationClient.ts

```diff
@@ -1542,4 +1557,227 @@ export class MetricsAdvisorAdministrationClient {
       span.end();
     }
   }
+
+  /**
+   * Creates data source credential for the given id
+   * @param datasourceCredential - the credential entity object to create
+   * @param options - The options parameter
+   */
+
+  public async createDatasourceCredential(
+    datasourceCredential: DatasourceCredentialUnion,
+    options: OperationOptions = {}
+  ): Promise<GetCredentialEntityResponse> {
+    const { span, updatedOptions: finalOptions } = createSpan(
+      "MetricsAdvisorAdministrationClient-createDatasourceCredential",
+      options
+    );
+    try {
+      // @ts-ignore
```

Comment:
what is this ignoring? is this a left over?

Reply:
removed